tests/cnf/ran/diskencryption/tests/tpm2.go

```diff
@@ -0,0 +1,358 @@
+
+		By("checking that Root disk is encrypted with tpm2 with PCR 1 and 7")
+		var luksListOutput string
+		luksListOutput, err = helper.GetClevisLuksListOutput()
+		Expect(err).ToNot(HaveOccurred(), "error getting of clevis luks list command")
+		isRootDiskTPM2PCR1AND7 := parsehelper.LuksListContainsPCR1And7(luksListOutput)
+		if !isRootDiskTPM2PCR1AND7 {
+			Skip("Root disk is not encrypted with PCR 1 and 7, skip TPM1 tests")
+		}
+
+		By("checking if Secure Boot is enabled")
+		var isSecureBootEnabled bool
+
+		Eventually(func() error {
+			isSecureBootEnabled, err = BMCClient.IsSecureBootEnabled()
+
+			return err
+		}).WithTimeout(tsparams.TimeoutWaitingOnBMC).
+			WithPolling(tsparams.PoolingIntervalBMC).ShouldNot(HaveOccurred(),
+			"IsSecureBootEnabled should not return an error")
+
+		if !isSecureBootEnabled {
+			Skip("Secure boot is not enabled test cannot proceed")
+		}
+
+		By("checking that the reserved slot is not present")
+		var isRootDiskReservedSlotPresent bool
+		luksListOutput, err = helper.GetClevisLuksListOutput()
+		Expect(err).ToNot(HaveOccurred(), "error getting of clevis luks list command")
+
+		isRootDiskReservedSlotPresent = parsehelper.LuksListContainsReservedSlot(luksListOutput)
+		Expect(err).ToNot(HaveOccurred(), "error checking if the output of clevis luks list contains a reserved slot")
+		Expect(isRootDiskReservedSlotPresent).To(BeFalse(), "there should be no reserved slot present at this point")
+
+		By("changing the server boot order")
+		swapFirstSecondBootItems()
+
+		By("restarting node gracefully")
+		err = cluster.SoftRebootSNO()
+		Expect(err).ToNot(HaveOccurred(), "error rebooting node")
+
+		By("waiting for pcr-rebind-boot log to appear (disk decryption succeeded)")
+		matches := []stdinmatcher.Matcher{
+			{
+				Regex: regexp.MustCompile("TPM failed"),
+				Times: 10,
+			},
+			{
+				Regex: regexp.MustCompile("pcr-rebind-boot"),
+				Times: 1,
+			},
+		}
+
+		ctx, cancel := context.WithTimeout(context.TODO(), tsparams.TimeoutWaitRegex)
+		defer cancel()
+		matchIndex, err := stdinmatcher.WaitForRegex(ctx, BMCClient, matches)
+
+		Expect(err).ToNot(HaveOccurred(), "WaitForRegex should not fail")
+		Expect(matchIndex).To(Equal(0), "WaitForRegex should match TPM failed (0)")
+
+		By("changing the server boot order back to defaults")
+		swapFirstSecondBootItems()
+
+		By("power cycling node")
+		Eventually(BMCClient.SystemPowerCycle).WithTimeout(tsparams.TimeoutWaitingOnBMC).
+			WithPolling(tsparams.PoolingIntervalBMC).ShouldNot(HaveOccurred(),
+			"power cycling the node should not return an error")
+
+	})
+})
+
+func swapFirstSecondBootItems() {
+	By("changing the server boot order back to defaults")
+
+	var bootRefs []string
+
+	var err error
+
+	Eventually(func() error {
+		bootRefs, err = BMCClient.SystemBootOrderReferences()
+
+		return err
+	}).ShouldNot(HaveOccurred(), "getting boot order should not return an error")
+
+	glog.V(ranparam.LogLevel).Infof("Current boot Order: %s\n", bootRefs)
```

Comment:
Instead of using `ranparam.LogLevel`, add this to `tsparams/const.go` and use `tsparams.LogLevel`:
```go
// LogLevel is the verbosity of glog statements in this test suite.
LogLevel glog.Level = 90
```

Reply:
fixed